api/controllers/game/spectate/join.js

```diff
@@ -24,7 +24,7 @@ module.exports = async function (req, res) {
     // Can't spectate if you're a player in an ongoing game
     const { p0, p1 } = game;
     if (game.status === GameStatus.STARTED && [ p0?.id, p1?.id ].includes(spectator.id)) {
-      return res.forbidden({ message: 'home.snackbar.cannotSpectate' });
+      return res.status(409).json({ message: 'home.snackbar.cannotSpectate' });
```

Comment:
Switched from forbidden to conflict; I think this better captures the spirit of the issue since it's really "you can't do that right now".